CHANGELOG.md

```diff
@@ -1,217 +1,242 @@
 
-[apollo client 3.0]: https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/
+[extensibility framework]: <https://pwastudio.io/pwa-buildpack/extensibility-framework/>
 
-### Complete cart and checkout experience
+### PWA Studio extensions
 
-This release finishes the full page cart and checkout features introduced in previous releases.
-The complete cart and checkout workflow is based on research made by members of the UX team.
+PWA Studio's extensibility framework lets developers create extensions and install them as project dependencies in their storefronts.
+As part of the work on the new extensibility framework, we refactored and relocated existing Venia features into PWA Studio extensions.
+We also developed new extensions that provide useful Venia features that developers can add to their projects.
 
-Storefront developers can use this streamlined process as implemented or they can use the different components to customize their own cart and checkout workflow.
+The source code for these extensions are available under the [`packages/extensions`][] directory in the PWA Studio repository.
 
-#### Shopping Bag feature
+`upward-security-headers`
+: intercepts build targets to add security headers to UPWARD
 
-In addition to the complete full page cart and checkout experience, this release introduces a new Mini-Cart/Shopping Bag feature.
+`venia-adobe-data-layer`
+: provides [Adobe Client Data Layer][] support for your project
 
-This feature is a floating modal that appears when you click on the shopping bag icon.
-It replaces the old `MiniCart` component, which previously appeared as a drawer from the right side of the app.
+`venia-sample-backends`
+: provides demo Magento backends and backend validation utilities for your project
+  (this extension should be removed prior to going live)
 
-Instead of competing with the full page cart feature, it only contains a subset of actions, such as removing an item and checking out.
-For additional modifications to cart products, it links to the cart page.
+`venia-sample-language-packs`
+: provides example translations to illustrate how new languages can be installed into your storefronts
 
-Developers can still use the old `MiniCart` component in their projects, but it has been renamed to `LegacyMiniCart`.
+[`packages/extensions`]: <https://github.com/magento/pwa-studio/tree/release/9.0/packages/extensions>
+[adobe client data layer]: <https://github.com/adobe/adobe-client-data-layer/wiki>
 
-### Branding updates
+### Internationalization and localization
 
-The UX team continues in their research to improve the look and feel of the Venia brand.
-This release includes many style updates to give the Venia storefront a more modern and accessible experience.
+The Internationalization(I18n) feature in PWA Studio lets developers localize their storefront content according to different regions and languages.
```

Comment:
I never normally see i18n with a capitalized I, doesn't really matter but consistency would be ideal. Also I'd lower case internationalization